system-contracts/test/precompiles/EcAdd.spec.ts

```diff
@@ -1,188 +0,0 @@
-import { expect } from "chai";
-import type { Contract } from "zksync-web3";
-import { callFallback, deployContractYul } from "../shared/utils";
-
-describe("EcAdd tests", function () {
```

Comment:
Why?...

Reply:
You should reuse testcases and redesign test rather than removing it

Reply:
I'll try to think of something, but it is not possible to tests end-to-end like in previous version. So here the implementation actually won't be tested, just general flow with mocked precompile call